Added config changes for MSA build along with one MSA function

vikram-dattu opened the pull request:
**Added config changes for MSA build along with one MSA function**
1. Changes in configure.ac and acinclude.m4 for MSA build
2. Modified BUILDING.md with cross-compile for MIPS procedure
3. Added one MSA optimized function: idct_islow_msa

Signed-off-by: Vikram Dattu vikram.dattu@imgtec.com

vikram-dattu committed:
Added config changes for MSA build along with one MSA function
1. Changes in configure.ac and acinclude.m4 for MSA build
2. Modified BUILDING.md with cross-compile for MIPS procedure
3. Added one MSA optimized function: idct_islow_msa

Signed-off-by: Vikram Dattu <vikram.dattu@imgtec.com>

ksraste commented:
1. Please remove "-funroll-loops" from BUILDING.md
2. Remove dead/unused macros from jmacros_msa.h
3. Check requirement of "upsample1_ptr" function and "my_upsampler" structure.
4. Remove "mips_idct_ifast_coefs" array if not used

vikram-dattu committed:
Incorporated review comments
1. Removed "-funroll-loops" from BUILDING.md
2. Removed unused macros from jmacros_msa.h
3. Removed unused structures/functions from jsimd_msa.c

Signed-off-by: Vikram Dattu <vikram.dattu@imgtec.com>
